app/src/main/res/layout/tab_tray_item.xml

```diff
@@ -0,0 +1,83 @@
+    <androidx.cardview.widget.CardView
+        android:id="@+id/mozac_browser_tabstray_card"
+        android:layout_width="100dp"
+        android:layout_height="56dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginStart="8dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:cardBackgroundColor="@color/photonWhite">
+
+        <ImageView
+            android:id="@+id/mozac_browser_tabstray_icon"
+            android:layout_width="50dp"
+            android:layout_height="28dp"
+            android:layout_marginTop="14dp"
+            android:layout_marginStart="25dp"
+            android:importantForAccessibility="no" />
+
+        <mozilla.components.browser.tabstray.thumbnail.TabThumbnailView
+            android:id="@+id/mozac_browser_tabstray_thumbnail"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:contentDescription="@string/mozac_browser_tabstray_open_tab" />
+
+    </androidx.cardview.widget.CardView>
+
+    <TextView
+        android:id="@+id/mozac_browser_tabstray_title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:ellipsize="end"
+        android:lines="1"
+        android:textSize="16sp"
+        android:paddingTop="16dp"
+        android:paddingStart="8dp"
+        android:paddingEnd="8dp"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintEnd_toStartOf="@id/mozac_browser_tabstray_close"
+        app:layout_constraintStart_toEndOf="@id/mozac_browser_tabstray_card"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/mozac_browser_tabstray_url"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="16dp"
+        android:layout_alignParentTop="true"
+        android:ellipsize="end"
+        android:lines="1"
+        android:paddingStart="8dp"
+        android:paddingEnd="8dp"
+        android:textColor="@color/tab_tray_item_selected_text_normal_theme"
+        android:alpha="0.50"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintEnd_toStartOf="@id/mozac_browser_tabstray_close"
+        app:layout_constraintStart_toEndOf="@id/mozac_browser_tabstray_card"
+        app:layout_constraintTop_toBottomOf="@id/mozac_browser_tabstray_title" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/mozac_browser_tabstray_close"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
+        android:layout_marginTop="24dp"
+        android:layout_marginEnd="16dp"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:contentDescription="@string/close_tab"
```

Comment:
we can follow up on this but wondering if the content description should be more descriptive to the tab it would be closing 🤔

Reply:
good idea, let's move these to wip/static strings so we're not exposing them to the l10n team